src/coreclr/src/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -34,6 +34,7 @@ The .NET Foundation licenses this file to you under the MIT license.
     <IlcGenerateCompleteTypeMetadata Condition="$(IlcGenerateCompleteTypeMetadata) == ''">true</IlcGenerateCompleteTypeMetadata>
     <IlcGenerateStackTraceData Condition="$(IlcGenerateStackTraceData) == ''">true</IlcGenerateStackTraceData>
     <IlcScanReflection Condition="$(IlcScanReflection) == ''">true</IlcScanReflection>
+    <SuppressTrimAnalysisWarnings Condition="$(SuppressTrimAnalysisWarnings) == ''">true</SuppressTrimAnalysisWarnings>
```

Comment:
@MichalStrehovsky This does not seem to be working. I am seeing a pile of warnings when building samples/HelloWorld using the current package.

Reply:
Sigh, I should have retested after applying the code review feedback. #397